packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,18 @@
+:local {
+  /* stylelint-disable selector-class-pattern  */
+  .card-container {
+    background-color: var(--terra-card-background-color, #fff);
+    border-radius: var(--terra-card-border-radius, 3px);
+    box-shadow: var(--tera-card-box-shadow, 0 1px 1px 0 #1c1f21);
+  }
```

Comment:
@mhemesath 
https://github.com/cerner/terra-core/pull/727#discussion_r133300635
screenshots and discussion on this link